internal/node_install.bzl

```diff
@@ -48,6 +41,8 @@ alias(name = "npm", actual = "bin/npm")
         stripPrefix = "node-v6.10.2-win-x64",
         sha256 = "d778ed84685c6604192cfcf40192004e27fb11c9e65c3ce4b283d90703b4192c"
     )
+    node = "node.exe"
```

Comment:
why is this `bin/node` above, but just `node.exe` here? Add a comment?

Reply:
adding comment